.github/workflows/health-check.yml

```diff
@@ -0,0 +1,31 @@
+name: Health Check
+
+on:
+  schedule:
+    # At minute 0 past hour 0800 and 2000.
+    - cron:  '0 8,20 * * *'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '8.x', '10.x', '12.x', '14.x' ]
```

Comment:
Not sure we need to run on node 8 & 10

Reply:
@JAdshead we would have to make a breaking change to drop it https://github.com/americanexpress/purgecss-loader/blob/master/.travis.yml#L3-L6